### ISSUE TITLE
[Bug][a11y] Consider increasing some elements from ETP-Exceptions page

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/fenix#16394]
## Steps to reproduce 
1. Have the Accessibility scanner app installed and opened.
2. Go to Settings - Privacy and security - ETP- Exceptions and scan the page.
3. Open a page and stop the tracking protection, making the page an exception.
4. Go back to Settings - Privacy and security - ETP- Exceptions and scan the page.

### Expected behavior
No suggestions are made.

### Actual behavior
Consider making the "Learn more" item larger, from 30 dp, to 48 dp or larger.
Consider making the close buttons from exceptions larger, from 32 dp x 32 dp, to 48 dp x 48 dp.

### Device information
* Android device: Google Pixel (Android 10)
* Fenix version: 11/3 Nightly
 
![exception](https://user-images.githubusercontent.com/40694158/98232353-3b7b6580-1f66-11eb-84da-f56ba84a4efb.png)

**Comment by etienne-chrp** [comment]
I would like to work on this issue, is it still unassigned?

Should be about updating the elements in: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/component_exceptions.xml

**Comment by fevziomurtekin** [comment]
#17516

**Comment by softvision-miralobontiu** [comment]
Sorry for the delay, the notification got lost in the numerous emails received.
I used the Accessibility scanner on the 2/15 Nightly build with Google Pixel (Android 10).
The first issue, the "Learn more" link isn't suggested to be increased anymore.
The "x" close button from the exceptions is suggested to be larger, from 32 dp x 32 dp, to 48 dp x 48 dp.
